pkiinit/option/optionHelpers.go

```diff
@@ -118,3 +124,32 @@ func getPkiCacheDirEnv() string {
 	}
 	return pkiCacheDir
 }
+
+func deploy(srcDir, destDir string) error {
+	return copyDir(srcDir, destDir)
+}
+
+func secureEraseFile(fileToErase string) error {
+	// grant the file read-write permission first
+	os.Chmod(fileToErase, os.ModePerm)
+	fileHdl, err := os.OpenFile(fileToErase, os.O_RDWR, 0666)
+	defer fileHdl.Close()
+	if err != nil {
+		return err
+	}
+	fileInfo, err := fileHdl.Stat()
+	if err != nil {
+		return err
+	}
+
+	fileSize := fileInfo.Size()
+	zeroBytes := make([]byte, fileSize)
+	copy(zeroBytes[:], "0")
+
+	// fill the file with zero bytes
+	if _, err := fileHdl.Write(zeroBytes); err != nil {
+		return err
+	}
+
+	return os.Remove(fileToErase)
```

Comment:
Looks like os.Remove might get called before fileHdl.Close().  Probably should close all open file handles before removing the file.